.appveyor.yml

```diff
@@ -106,6 +106,12 @@ environment:
     - PROJ: "onnxruntime"
       OS: windows-x86_64
       EXT: -gpu
+    - PROJ: "ffmpeg"
+      OS: windows-x86
+      EXT: -gpl
+    - PROJ: "ffmpeg"
+      OS: windows-x86_64
+      EXT: -gpl
```

Comment:
Let's put those next to their default builds, like you did in `.travis.yml`.

Reply:
Fixed.